misc/performance/nydusd_config.json

```diff
@@ -0,0 +1,26 @@
+{
+    "device": {
+        "backend": {
+            "type": "registry",
+            "config": {
+                "scheme": "http",
+                "host": "localhost:5000"
+            }
+        },
+        "cache": {
+            "type": "blobcache",
+            "config": {
+                "work_dir": "/var/lib/containerd-nydus/cache"
+            }
+        }
+    },
+    "mode": "direct",
+    "digest_validate": false,
+    "enable_xattr": true,
+    "iostats_files": true,
```

Comment:
Let's set these metric options to false: `iostats_files`, `access_pattern`, `latest_read_files`.